lacework/resource_lacework_alert_channel_victorops.go

```diff
@@ -193,31 +193,4 @@ func resourceLaceworkAlertChannelVictorOpsDelete(d *schema.ResourceData, meta in
 
 	log.Printf("[INFO] Deleted %s integration with guid %s\n", api.VictorOpsAlertChannelType, d.Id())
 	return nil
-}
-
-func validateVictorOpsAlertChannelResponse(response *api.VictorOpsAlertChannelResponse) error {
```

Comment:
I think we forgot to remove this code. cc @dmurray-lacework I removed it for you 😉